screens/SignUpScreen.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+
+import {
+    Container,
```

Comment:
this is for @sonamdkindy it's kinda hard to keep track of all of the imports... we could add eslint to have it notify us if we import something that isn't used.